### PR TITLE
Configure testnet

### PR DESCRIPTION
**Pull request by justinengland**
this change allows users to swap to the current production testnet "testnet7" without pulling a specific branch. 

And can be easily upgrade to new testnets as needed.

### CODE REVIEW

File: chia/cmds/configure.py [hunk 1]
```diff
@@ -71,13 +72,37 @@ def configure(
...
+        testnet_port = "58444"
+        testnet_introducer = "beta1_introducer.chia.net"
+        testnet = "testnet7"
+        config["full_node"]["port"] = int(testnet_port)
+        config["full_node"]["introducer_peer"]["port"] = int(testnet_port)
+        config["farmer"]["full_node_peer"]["port"] = int(testnet_port)
+        config["timelord"]["full_node_peer"]["port"] = int(testnet_port)
+        config["wallet"]["full_node_peer"]["port"] = int(testnet_port)
+        config["wallet"]["introducer_peer"]["port"] = int(testnet_port)
+        config["introducer"]["port"] = int("58444")
```
Comment (cmmarslender) [inline chia/cmds/configure.py:85]: to be consistent with the rest?
```suggestion
        config["introducer"]["port"] = int(testnet_port)
```

File: chia/cmds/configure.py [hunk 2]
```diff
@@ -71,13 +72,37 @@ def configure(
...
+        config["full_node"]["introducer_peer"]["host"] = testnet_introducer
+        config["selected_network"] = testnet
+        config["harvester"]["selected_network"] = testnet
```
Comment (cmmarslender) [inline chia/cmds/configure.py:88]: I think all these `testnet` values should probably be quoted unless theres a testnet var somewhere I missed at first glance?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: chia/cmds/configure.py [hunk 3]
```diff
@@ -71,13 +72,39 @@ def configure(
         else:
             print("uPnP disabled")
         change_made = True
+    if testnet is not None:
```
Comment (aqk) [inline chia/cmds/configure.py:75]: Question: is there another config variable that might conflict with this one, e.g. a "mainnet" variable?

**Review by aqk — APPROVED** [review]

**Review by cmmarslender — APPROVED** [review]

**Review by atyhieu — COMMENTED** [review]
Ok

**Pull request merged**